core/strings/conversion.odin

```diff
@@ -57,13 +71,20 @@ to_valid_utf8 :: proc(s, replacement: string, allocator := context.allocator) ->
 	}
 	return to_string(b)
 }
-
 /*
-	returns the input string `s` with all runes set to lowered case
-	always allocates using the `allocator`
+Converts the input string `s` to all lowercase characters.
 
-	strings.to_lower("test") -> test	
-	strings.to_lower("Test") -> test	
+*Allocates Using Provided Allocator*
+
+Inputs:
+- s: Input string to be converted.
+- allocator: (default: context.allocator).
+
+Example:
+```odin
+	strings.to_lower("TeST") -> test
+```
```

Comment:
The ticks and odin is not required. The website already knows that odin docs are being generated. Only a tab is required and markdown will format it also properly. You should also make all your examples fully self contained.
i.e. (note that tabs are significant here)
```
Example:

	import "core:strings"
	import "core:fmt"
	import "core:mem"

	strings_to_lower_example :: proc() -> (err: mem.Allocator_Error) {
		s :: "TeST"
		s_lower_cased := strings.to_lower(s, context.allocator) or_return
		fmt.println(s_lower_cased)
	}

Output:

	test

```

The `Output:` is not necessary but the website will properly format and test it soon so would be good to start.

Reply:
Not a rule of thumbs but it will be easier to generate specific things if the example procedure name follows the form
`<Package_Name>_<Procedure_Name>_example`